assets/helpers/payPalExpressCheckout/payPalExpressCheckout.js

```diff
@@ -71,71 +73,63 @@ function setupPayment(dispatch: Function, state: CombinedState) {
         if (token) {
           resolve(token.token);
         } else {
-          dispatch(payPalExpressError('PayPal token came back blank.'));
```

Comment:
Were we previously just doing nothing with this action?

Reply:
Not as far as I can tell: https://github.com/guardian/support-frontend/search?utf8=%E2%9C%93&q=PAYPAL_EXPRESS_ERROR&type=

Reply:
With this PR any errors should be going to Sentry (although once we start seeing them I expect some further refinement will be needed)...